composable-tasks/src/compose_tasks.rs

```diff
@@ -20,6 +20,7 @@ pub enum TaskType {
     UnwrapEGLD,
     Swap,
     RouterSwap,
+    SmartSwap,
```

Comment:
Although it looks nicer this way, put "SmartSwap" as the last task type. Otherwise, you could run into problems with different versions of `TaskType`.